### DISCUSSION TITLE
ait-wiki/mechanics/artron/

### DISCUSSION THREAD

**Discussion by giscus** [amblelabs/ait-wiki#2] [General]
# ait-wiki/mechanics/artron/

⚠️ This article will be split into sub-articles for each given block or item. What Is It? Artron Energy is the fuel the TARDIS runs on. It is known in the TARDIS itself as “AU”, or just simply as a percentage. It comes from rifts in Overworld, or passively anywhere.
Where Is It Collected From? Artron Energy is collected by the TARDIS during refueling, and can be passively gained from anywhere (slowly) or gained in larger amounts quicker in a rift chunk.

https://theo.is-a.dev/ait-wiki/mechanics/artron/

**Comment by MaketendoDev** [comment]
i liek eating artron energy yum
